spring-cloud-gcp-samples/pom.xml

```diff
@@ -166,7 +166,7 @@
 								<gcs-local-directory>/tmp/gcp_integration_tests/integration_storage_sample</gcs-local-directory>
 								<spring.cloud.gcp.sql.instance-connection-name>spring-cloud-gcp-ci:us-central1:testmysql</spring.cloud.gcp.sql.instance-connection-name>
 								<spring.cloud.gcp.sql.database-name>code_samples_test_db</spring.cloud.gcp.sql.database-name>
-								<spring.datasource.password>test</spring.datasource.password>
+								<spring.datasource.password>${spring-cloud-gcp-ci-mysql-db-root-password}</spring.datasource.password>
 								<spring.r2dbc.password>test</spring.r2dbc.password>
```

Comment:
Do we have to set the password at all? AFAIK we're using integration authentication which is safe without a db password. Also, wouldn't we have to fix the r2dbc password as well?

Reply:
This is intended to fix a vulnerability brought by [b/296915631](b/296915631). 
I'm currently testing this config to determine where we need to add this password

Reply:
Added this to PR description as well

Reply:
I converted this to a draft in the meantime